src/storage/IndexedDBStorage.ts

```diff
@@ -343,32 +348,32 @@ export default class IndexedDBStorage<T> extends InMemoryStorage<T> {
 			resolveInitialPromise = resolve;
 			rejectInitialPromise = reject;
 		});
-		this.dbName = options.dbName || 'store-database';
-		this.objectStoreName = options.objectStoreName || 'items';
-		this.dbInitializationPromise = initializationPromise;
-		this.indices = new Set<keyof T>();
+		this._dbName = options.dbName || 'store-database';
+		this._objectStoreName = options.objectStoreName || 'items';
+		this._dbInitializationPromise = initializationPromise;
+		this._indices = new Set<keyof T>();
```

Comment:
nit: properties like `this._indices` can be initialised in place - `private _indices = new Set<string>`.